### PR TITLE
Outline Components to Nubs

### PR DESCRIPTION
**Pull request by seeratawan01**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

<a href="https://gitpod.io/#https://github.com/Hakkei-Co/nub-ui/pull/3"></a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -121,6 +121,7 @@
     "@storybook/addon-a11y": "^6.4.19",
     "@storybook/addon-actions": "^6.4.19",
     "@storybook/addon-controls": "^6.4.19",
+    "@storybook/addon-docs": "^6.4.20",
```
Comment (laujonat) [inline package.json]: I think addon-controls deprecates addon-docs.  Was there a reason why you added this?

File: src/resolved-outline-config.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/**
+ * This file is auto generated via npm run generate.resolved-outline
```
Comment (laujonat) [inline src/resolved-outline-config.ts]: Where did this come from?

**Review by laujonat — COMMENTED** [review]
This looks great! Nice work 👍 

Let's try and figure out why the build is failing.

File: src/resolved-outline-config.ts (hunk 2 above)
Comment (seeratawan01) [reply src/resolved-outline-config.ts]: This is auto generated file, I will remove it and add it to .gitignore

File: package.json (hunk 1 above)
Comment (seeratawan01) [reply package.json]: I was getting some mdx errors from storybook, added this for testing.

**Comment by seeratawan01** [comment]
Rollup-plugin-summary is causing the build failure, so I commented it in rollup config file and it's working fine.

**Review by laujonat — APPROVED** [review]
great work @seeratawan01

File: .gitignore [hunk 3]
```diff
@@ -32,3 +32,4 @@ Thumbs.db
 UserInterfaceState.xcuserstate
 src/component-list.json
 src/resolved-nubs-config.ts
+src/resolved-outline-config.ts
```
Comment (laujonat) [inline .gitignore:35]: why is this added? @seeratawan01

Comment (seeratawan01) [reply .gitignore:35]: resolved-outline-config.ts got auto generated when I was first settingup the project, so to be on save side I added it in .gitignore file to not push this file (resolved-outline-config.ts) into the repo if it ever get generated in future again.
